phivk/gradu.al
# Feature/adds data generation script

This is adding the script that will be run to generate the data. This fetches from a Google Sheet, formats the data and places it in the correct directory. Once we're happy with how this works, we will look to automate the execution of this.

To test:
- Run yarn i
- Delete the `/content/storytellersunited/data` content
- Run `node fetchData` from the root of the directory
- Confirm that the data is repopulated.
